packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -554,13 +554,30 @@ export namespace EmrCreateCluster {
     TERMINATE_CLUSTER = 'TERMINATE_CLUSTER',
   }
 
+  /**
+   * Spot Allocation Strategies
+   *
```

Comment:
add detail here about what these are (can copy/paste from AWS docs) and link to documentation